docs/Model/v2/UpdateInventoryCountryValuesRequest.md

```diff
@@ -10,8 +10,10 @@ Name | Type | Description | Notes
 **return_shipping_costs** | **float** | Return shipping Cost | [optional] 
 **delivery_min_time** | **int** | Minimal Delivery Time | 
 **delivery_max_time** | **int** | Maximum Delivery Time | 
-**reverse_charge** | **int** | Reverse charge | [optional] 
+**reverse_charge** | **int** | Reverse charge. If unknown use &#39;0&#39; as value. | 
```

Comment:
Is this field mandatory now? If so: will it break backwards compatibility? 

It seems like this is still optional, since we default it to `null` when creating and updating. So I think we should add optional here, or should we? 🙂
```suggestion
**reverse_charge** | **int** | Reverse charge. If unknown use &#39;0&#39; as value. | [optional]
```

Reply:
The API was already changed a year ago, as you can see here:
https://gitlab.real-digital.de/code/sms/-/commit/0beee305073ce09700d4770631d399cf2f9272b3
So it is more about getting the SDK back on track with the API.